personnel.md

```diff
@@ -0,0 +1,216 @@
+# Personnel
+
+## Current
+
+### Secretaries
+
+| Name                                  | Term                    |
+|---------------------------------------|-------------------------|
+| Alessandro Lai ([@AlessandroLai])     | 2017-11-12 - 2018-08-31 |
+| Margret Staples ([@dead_lugosi])      | 2017-11-12 - 2019-05-31 |
+| Michael Cullum ([@michaelcullumuk])   | 2016-01-31 - 2018-01-31 |
```

Comment:
We normally list from oldest to newest to be consistent.

Reply:
Oldest to newest referring to the start or the end of the term?

Reply:
Start

Reply:
Stupid question, ignore it :stuck_out_tongue_closed_eyes: